.github/workflows/build.yml

```diff
@@ -29,6 +24,15 @@ jobs:
       - name: Run lint
         run: |
           make lint
+      - name: Install and start Redis
+        run: |
+          make testdeps
+          make start-redis
```

Comment:
Do we need to start redis here? I think this can be handled by the test suite with `startRedisServer`. It's probably better to do it in code, we may have more control over the configuration.